src/main/java/art/heredium/controller/admin/AdminCompanyMembershipController.java

```diff
@@ -0,0 +1,33 @@
+package art.heredium.controller.admin;
+
+import java.io.IOException;
+
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/admin/company-membership")
```

Comment:
```suggestion
@RequestMapping("/api/file/template/company-membership")
```
Đáp mớ này sang FileController.

Reply:
I updated in https://github.com/Incuvers-IT/heredium-BE/pull/80/commits/8c4ea2a1e5a773251925faacaa876f7bcbcba37c